.travis.yml

```diff
@@ -10,6 +10,7 @@ script:
 dist: trusty
 sudo: required
 env:
+  - TEST_SUITE=haml
```

Comment:
This is getting out of hand. We should not have a test suite for every linter we do. How about we integrate all of them into on suite?